entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java

```diff
@@ -103,7 +104,7 @@ protected String serialize(EntityRecord entityRecord, FormatTypes format, String
      * @throws EntityCreationException
      */
     public ResponseEntity<String> generateResponseEntity(String profile, FormatTypes outFormat,
-                                                         String contentType, EntityRecord entityRecord, HttpStatus status)
+                                                         String contentType, EntityRecord entityRecord, HttpStatus status, RequestMethod requestMethod)
```

Comment:
contentType is redundant, the FormatTypes outFormat defines the serilization and the content type header as well

Reply:
Improved.